experiments/imit_benchmark.sh

```diff
@@ -72,9 +73,10 @@ parallel -j 25% --header : --results ${LOG_ROOT}/parallel/ --colsep , --progress
   ${ALGORITHM} \
   ${extra_configs} \
   {env_config_name} \
-  log_dir="${LOG_ROOT}/${ALGORITHM}/{env_config_name}_{seed}" \
+  log_dir="${LOG_ROOT}/{env_config_name}_{seed}/n_expert_demos_{n_expert_demos}" \
```

Comment:
Why do we not want `${ALGORITHM}` any longer? I guess this is not needed given log directory is unique thanks to timestamping (modulo race conditions).

Reply:
The directory that was being uploaded to S3 ending up looking like `data/gail/gail`. Might be better to make a fix in the `aws s3` command instead. Taking a look...

Reply:
Probably a trailing `/` where there shouldn't be one (or vice versa), I've had this kind of problem with `rsync` before

Reply:
`${ALGORITHM}` was never necessary for uniqueness (minus race conditions, yeah). There was also a problem where both this line and the S3 sync command explicitly appended `${ALGORITHM}/` to the path. To fix the double path `gail/gail`, I removed the `${ALGORITHM}/` from this line.